test-requirements.txt

```diff
@@ -1 +1,2 @@
 odoo-test-helper
+odoo-addon-account_statement_import_camt @ git+https://github.com/OCA/bank-statement-import.git@refs/pull/684/head#subdirectory=account_statement_import_camt
```

Comment:
```suggestion
```
can be removed as https://github.com/OCA/bank-statement-import/pull/684 is now merged :tada: